spec/services/client_api/teams_service_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe ClientApi::TeamsService do
+  let(:team_one) { create :team }
+  let(:user_one) { create :user }
+
+  it 'should raise an ClientApi::CustomTeamError if user is not assigned' do
+    expect {
+      ClientApi::TeamsService.new(team_id: team_one.id)
```

Comment:
maybe add this to local variable

Reply:
if you rise an error outside of expect block the test blows up :smile: 
but this PR has a label WIP which mean that is work in progress and is not finished jet :smile: